### PR TITLE
[fuchsia] Point TODOs off closed bug.

### PR DESCRIPTION
**Pull request by akbiggs**

### CODE REVIEW

File: shell/platform/fuchsia/flutter/component_v2.cc [hunk 1]
```diff
@@ -254,11 +254,11 @@ ComponentV2::ComponentV2(
   cloned_directory_ptr_.set_error_handler(
       [this](zx_status_t status) { cloned_directory_ptr_.Unbind(); });
 
-  // TODO(fxb/50694): Close handles from ComponentStartInfo::numbered_handles
+  // TODO(akbiggs): Close handles from ComponentStartInfo::numbered_handles
```
Comment (arbreng) [inline shell/platform/fuchsia/flutter/component_v2.cc:257]: Create a bug for the 3 TODO(akbiggs)?  Even something like "flutter_runner CFv2 cleanup"

**Review by arbreng — APPROVED** [review]
LGTM after filing bug

**Comment by chinmaygarde** [comment]
I suspect this isn't good to land in this state right?

File: shell/platform/fuchsia/flutter/component_v2.cc (hunk 1 above)
Comment (akbiggs) [reply shell/platform/fuchsia/flutter/component_v2.cc]: Done.

**Review by arbreng — APPROVED** [review]

**Pull request merged**